src/Database/Driver/Mysql.php

```diff
@@ -75,6 +75,13 @@ class Mysql extends Driver
      */
     protected $_supportsNativeJson;
 
+    /**
+     * Whether or not the connected server supports windows.
+     *
+     * @var bool
+     */
+    protected $_supportsWindows;
```

Comment:
Can we name this `$_supportsWindowFunctions` instead?

Reply:
I was also like "wuuut?" :P

Reply:
Sure. I was just using the technical name for the expression, but can be confusing.